tests/unit_tests/test_unit.py

```
@@ -2461,6 +2461,53 @@ def test_spectrum_preprocessing(tmp_path, mgf_small):
+    )
+    # Local counts of 2 and 3 from two processes are summed.
+    model.all_gather = lambda x: x.new_tensor([2, 3])
+    model.n_missing_predictions = 2
+    model.on_predict_epoch_end()
+    assert model.n_missing_predictions == 5
+
+
+def test_log_annotate_report_missing_predictions(monkeypatch):
+    mock_logger = unittest.mock.MagicMock()
+    monkeypatch.setattr("casanovo.utils.logger", mock_logger)
+    monkeypatch.setattr(utils, "log_run_report", lambda **kw: None)
```

Ruff (0.16.1)

[warning] Unused lambda argument: `kw`

(ARG005)